src/main/kotlin/platform/forge/util/ForgeConstants.kt

```diff
@@ -51,6 +51,6 @@ object ForgeConstants {
     // From https://github.com/MinecraftForge/MinecraftForge/blob/38a5400a8c878fe39cd389e6d4f68619d2738b88/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModInfo.java#L45
     val MOD_ID_REGEX = "^[a-z][a-z0-9_-]{1,63}$".toRegex()
 
-    val DISPLAY_TEST_MANIFEST_VERSION = SemanticVersion.release(41, 0, 15)
+    val DISPLAY_TEST_MANIFEST_VERSION = SemanticVersion.release(40, 2, 14)
```

Comment:
Versions between 41.0.0 and 41.0.14 but I'll let that pass because this is a rather small range of old versions.